runtime/local/Cargo.toml

```diff
@@ -56,8 +56,8 @@ pallet-transaction-payment-rpc-runtime-api = { git = "https://github.com/parityt
 # Astar pallets
 pallet-block-reward = { git = "https://github.com/AstarNetwork/astar-frame", tag = "pallet-block-reward-2.0.0/polkadot-v0.9.17", default-features = false }
 pallet-custom-signatures = { git = "https://github.com/AstarNetwork/astar-frame", branch = "polkadot-v0.9.17", default-features = false }
-pallet-dapps-staking = { git = "https://github.com/AstarNetwork/astar-frame", tag = "pallet-dapps-staking-3.2.0/polkadot-v0.9.17", default-features = false }
-pallet-precompile-dapps-staking = { git = "https://github.com/AstarNetwork/astar-frame", tag = "pallet-dapps-staking-3.2.0/polkadot-v0.9.17", default-features = false }
+pallet-dapps-staking = { git = "https://github.com/AstarNetwork/astar-frame", tag = "pallet-dapps-staking-3.3.0/polkadot-v0.9.17", default-features = false }
```

Comment:
Bump up to `3.3.1` in both local and Shibuya. That will fix the precompiles problem.